TLM/TLM/UI/LegacySubTool.cs

```diff
@@ -185,6 +184,11 @@ public abstract class LegacySubTool : IObserver<ModUI.EventPublishers.UIOpacityN
         public virtual void ShowGUIOverlay(ToolMode toolMode, bool viewOnly) { }
 
         public virtual bool IsCursorInPanel() {
+            return IsCursorInPanel_();
+        }
+
+        /// <summary>Check whether cursor is in the main menu or debug menu area.</summary>
+        internal static bool IsCursorInPanel_() {
```

Comment:
yiu are going to remove one of the two later on right?

Reply:
The legacysubtool class will be deleted in the end of this.
I am merging smaller branches into this EPIC branch

Reply:
it would actually be better to have intermediate steps that can be merged directly into master though, each via its own PR, but ok.